compiler/src/test/java/com/squareup/anvil/compiler/InterfaceMergerTest.kt

```diff
@@ -622,4 +623,26 @@ class InterfaceMergerTest(
       assertThat(componentInterface extends secondContributingInterface).isTrue()
     }
   }
+
+  @Test fun `anonymous inner classes should not be evaluated`() {
```

Comment:
This case came up initially from an `InterfaceMerger`-based stacktrace, although technically this test isn't specific to `InterfaceMerger`. Open to giving it a new home if you have a better suggestion @vRallev

Reply:
I think that's fine.